.vscode/launch.json

```diff
@@ -0,0 +1,46 @@
+{
+    "configurations": [
+        {
+            "name": "micro:bit cortex debug",
+            "cwd": "${workspaceFolder}",
+            "executable": "${workspaceFolder}/build/MICROBIT",
+            "request": "launch",
+            "type": "cortex-debug",
+            "servertype": "pyocd",
+            "interface": "swd",
+            "device": "nrf52",
+            "targetId": "nrf52",
+            "runToMain": false,
+            "boardId": "9904",
+            "svdFile": "${workspaceFolder}/libraries/codal-nrf52/nrfx/mdk/nrf52833.svd",
+            "preLaunchCommands": [
+              "load",
+              "add-symbol-file ${workspaceFolder}/build/MICROBIT",
+              "enable breakpoint",
+              "monitor reset"
+            ]
+        },
+
+        {
+            "name": "micro:bit openOCD cortex debug",
+            "cwd": "${workspaceFolder}",
+            "executable": "${workspaceFolder}/build/MICROBIT",
+            "request": "launch",
+            "type": "cortex-debug",
+            "servertype": "openocd",
+            "configFiles": ["${workspaceFolder}/openocd.cfg"],

```

Comment:
Could this config file be replaced with `openOCDLaunchCommands`?

Reply:
Seems like that for additional things like setting the interface etc. It couldn't be used to replace everything. However, from looking around it seems we don't actually need `openocd.cfg` as you can just specify directly what's required nb

```
"configFiles": [
    "interface/cmsis-dap.cfg",
    "target/nrf52.cfg"
],
```
Done with a08ceb9.